File: tests/new_tests/access_class_value.jl

# @dataclass
struct Packet
    val::Float64
end


# @dataclass
struct Register
    PACKET::Packet	
    VALUE::Int64
end


function main()
    a = Register(Packet(1.3), 10)
    println(join(["OK"], " "));
end

main()

File: tests/new_tests/access_class_value.py
# On hold until @sealed is solved
from dataclasses import dataclass

@dataclass
class Packet:
    val: float

@dataclass
class Register:
    PACKET: Packet
    VALUE: int

if __name__ == "__main__":
    a = Register(Packet(1.3), 10)
    print("OK")

# Copied from 'cases' folder